db/migrate/20180228194754_add_ldap_config_to_settings.rb

```diff
@@ -0,0 +1,53 @@
+class AddLdapConfigToSettings < ActiveRecord::Migration[5.1]
+  def up
```

Comment:
There's a file `app/lib/default_settings_populator` with the logic to create existing settings. Ideally we should open ldap config and populate the settings there in case someone needs to re-generate these settings. Because the setting keys are unique, it can be rerun and won't overwrite existing settings but will add new settings if any are found.